apps/remix-ide/src/app/files/remixd-handle.js

```diff
@@ -130,25 +131,28 @@ export class RemixdHandle extends WebsocketPlugin {
 }
 
 function remixdDialog () {
-  const commandText = 'remixd -s absolute-path-to-the-shared-folder --remix-ide your-remix-ide-URL-instance'
+  const commandText = 'remixd -s path-to-the-shared-folder --remix-ide remix-ide-instance-URL'
   return yo`
     <div class=${css.dialog}>
       <div class=${css.dialogParagraph}>
-        Access your file system from Remix IDE. Remixd the NPM module needs to be running in the background to use the Remixd plugin. For more info please check the <a target="_blank" href="https://remix-ide.readthedocs.io/en/latest/remixd.html">Remixd tutorial</a>.
+        Access your local file system from Remix IDE using <a target="_blank" href="https://www.npmjs.com/package/@remix-project/remixd">Remixd NPM package</a>.<br/><br/> 
+        Remixd needs to be running in the background to load the files in localhost workspace. For more info, please check the <a target="_blank" href="https://remix-ide.readthedocs.io/en/latest/remixd.html">Remixd tutorial</a>.
       </div>
-      <div class=${css.dialogParagraph}>If you are just looking for the remixd command here it is:
-        <br><br><b>remixd -s absolute-path-to-the-shared-folder --remix-ide your-remix-ide-URL-instance</b>
+      <div class=${css.dialogParagraph}>If you are just looking for the remixd command, here it is:
+        <br><br><b>${commandText}</b>
         <span class="">${copyToClipboard(() => commandText)}</span>
       </div>
-      <div class=${css.dialogParagraph}>A connection will start a session between <em>${window.location.origin}</em> and your local file system <i>ws://127.0.0.1:65520</i>
-        <br>To see that a connection has been made, check that there is a localhost section in the Files Explorer
+      <div class=${css.dialogParagraph}>On successful connection, a session will be started between <em>${window.location.origin}</em> and your local file system at <i>ws://127.0.0.1:65520</i>
+         and files will be loaded under localhost workspace in the File Explorers. 
+        <br/>Please note, if shared folder is a hardhat project, an additional hardhat websocket plugin will be listening at <i>ws://127.0.0.1:65522</i>
```

Comment:
Note, if the shared folder is a Hardhat project, an additional Hardhat websocket plugin will be listening at

Reply:
updated